src/utils/designConstants.js

```diff
@@ -0,0 +1,22 @@
+export const BREAKPOINTS = {
+	s: '440px',
+	m: '640px',
+	l: '840px',
+	xl: '1024px'
+};
+
+export const MEDIA_QUERIES = {
+	small: `screen and (min-width: ${BREAKPOINTS['s']})`,
+	medium: `screen and (min-width: ${BREAKPOINTS['m']})`,
+	large: `screen and (min-width: ${BREAKPOINTS['l']})`,
+	huge: `screen and (min-width: ${BREAKPOINTS['xl']})`
```

Comment:
Not sure why the linter doesn't pick up the unnecessary brackets in these `BREAKPOINT` references, but they should all be dot-separated references, e.g. `BREAKPOINT.s`, `BREAKPOINT.m`, for style consistency with the rest of the codebase.